canonical-websites/www.ubuntu.com
# Move enterprise k8s strip

## Done

* moved enterprise strip up
* updated location of pdf to a redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [k8s](http://0.0.0.0:8001/containers/kubernetes)
- see the strip is after the banner, and links to a pdf at enterprise-canonical-kubernetes which links to https://assets.ubuntu.com/v1/Enterprise-Canonical-Kubernetes.pdf
